MAUI/TreeMap/right-to-left.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: Drilldown in .NET MAUI TreeMap Control | Syncfusion
```

Comment:
Add proper title - title: Drilldown in .NET MAUI TreeMap Control | Syncfusion

Reply:
changes updated